phpBB/phpbb/language/language_file_helper.php

```diff
@@ -15,6 +15,7 @@
 
 use DomainException;
 use phpbb\json\sanitizer as json_sanitizer;
+use phpbb\user;
```

Comment:
Seems that this created a circular reference via: path: **"user -> language -> language.helper.language_file -> user".**